profiles/api.py

```diff
@@ -256,3 +285,119 @@ def is_user_info_complete(user):
         and hasattr(user, "legal_address")
         and user.legal_address.is_complete
     )
+
+
+def import_alums(alums):
+    """
+    Import alums
+
+    Args:
+        alums (Alums namedtuple)
+    """
+    try:
+        user = User.objects.get(email=alums.learner_email)
+    except User.DoesNotExist:
+        user = User.objects.create(
+            email=alums.learner_email, username=alums.learner_email
+        )
+
+        from profiles.models import Profile, LegalAddress
+
+        LegalAddress.objects.create(user=user)
+        Profile.objects.create(user=user)
+        user.username = usernameify("", user.email)
+        user.save()
+
+        log.info(
+            "User does not exist against that email=%s but created without password and incomplete profile",
+            alums.learner_email,
+        )
+    bootcamp_start_date = datetime.strptime(alums.bootcamp_start_date, "%Y-%m-%d")
+    bootcamp_end_date = datetime.strptime(alums.bootcamp_end_date, "%Y-%m-%d")
+    if is_naive(bootcamp_start_date):
+        bootcamp_start_date = make_aware(bootcamp_start_date)
```

Comment:
Since we don't know the timezone, and since the date is the only important part anyway, we should set the timezone to UTC here and in the next line: `make_aware(bootcamp_start_date, tz=pytz.UTC)`.